docs/SalesforceStreaming-streamingsource.md

```diff
@@ -0,0 +1,91 @@
+This field not required, if you are using an existing push topic.
+
+
+**Notify On Create:** Push topic property, which specifies if a create operation should generate a record.
+
+
+**Notify On Update:** Push topic property, which specifies if a update operation should generate a record.
+
+
+**Notify On Delete:** Push topic property, which specifies if an delete operation should generate a record.
+
+
+**Notify For Fields:** Push topic property, which specifies how the record is evaluated against the
+PushTopic query The NotifyForFields values are:<br>
+_All_	- Notifications are generated for all record field changes, provided the evaluated records match
+the criteria specified in the WHERE clause.<br>
+_Referenced (default)_ -	Changes to fields referenced in the SELECT and WHERE clauses are evaluated.
+Notifications are generated for the evaluated records only if they match the criteria specified
+in the WHERE clause.<br>
+_Select_	- Changes to fields referenced in the SELECT clause are evaluated. Notifications are generated
+for the evaluated records only if they match the criteria specified in the WHERE clause.<br>
+_Where_	- Changes to fields referenced in the WHERE clause are evaluated. Notifications are generated
+for the evaluated records only if they match the criteria specified in the WHERE clause.
+
+**SObject Name:** Salesforce object name to read. If value is provided, plugin will get all fields for this object from
+Salesforce and generate SOQL query (`select <FIELD_1, FIELD_2, ..., FIELD_N> from ${sObjectName}`).
+Ignored if SOQL query is provided.
+
+**Error Handling:** Strategy used to handle erroneous records. Acceptable values are Skip on error, Stop on error.<br>
+Skip on error - Ignores erroneous records.<br>
+Stop on error - Fails pipeline due to erroneous record
+
+**Schema:** The schema of output objects.
+The Salesforce types will be automatically mapped to schema types as shown below:
+
+
++-------------+----------------------------------------------------------------------------+--------------+
+| Schema type |                              Salesforce type                               |    Notes     |
++-------------+----------------------------------------------------------------------------+--------------+
+| bool        | _bool                                                                      |              |
+| int         | _int                                                                       |              |
+| long        | _long                                                                      |              |
+| double      | _double, currency, percent                                                 |              |
+| date        | date                                                                       |              |
+| timestamp   | datetime                                                                   | Milliseconds |
```

Comment:
what is the difference between timestamp and time if they are both in milliseconds?

Reply:
Under the hood we are just using CDAP logical types
https://github.com/cdapio/cdap/blob/1d2d51a2e5c467891b9fe994133ef80646578a09/cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java#L105

It's TIME_MILLIS vs TIMESTAMP_MILLIS. The one has date and time the other only date.